bundles/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/internal/configuration/GPIOConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.gpio.internal.configuration;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link GPIOConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Nils Bauer - Initial contribution
+ */
+@NonNullByDefault
+public class GPIOConfiguration {
+
+    /**
+     * The id of the gpio pin.
+     */
+    public @Nullable Integer gpioId;
+
+    /**
+     * Should the input/output be inverted?
+     */
+    public Boolean invert = false;
```

Comment:
```suggestion
    public voolean invert = false;
```

use the primitive where possible